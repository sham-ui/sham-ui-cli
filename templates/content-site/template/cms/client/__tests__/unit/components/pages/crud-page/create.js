import PagesCrudPageCreate  from '../../../../../src/components/pages/crud-page/create.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( PagesCrudPageCreate, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
