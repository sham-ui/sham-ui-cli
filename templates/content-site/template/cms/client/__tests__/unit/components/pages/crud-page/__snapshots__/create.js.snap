// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "ID": "component",
    "onSuccess": [Function],
    "saveData": [Function],
    "typeName": "",
  },
  "html": "<!--0-->",
}
`;
